src/chocolatey/GetChocolatey.cs

```diff
@@ -538,8 +539,8 @@ private void ExtractResources()
             }
             catch (Exception ex)
             {
-                this.Log().Warn(ChocolateyLoggers.Important, "Please ensure that ChocolateyInstall environment variable is set properly and you've run once as an administrator to ensure all resources are extracted.");
-                this.Log().Error("Unable to extract resources. Please ensure the ChocolateyInstall environment variable is set properly. You may need to run once as an admin to ensure all resources are extracted. Details:{0} {1}".FormatWith(Environment.NewLine, ex.ToString()));
+                this.Log().Warn(ChocolateyLoggers.Important, "Please ensure that {0} environment variable is set properly and you've run once as an administrator to ensure all resources are extracted.", EnvironmentVariables.System.ChocolateyInstall);
+                this.Log().Error("Unable to extract resources. Please ensure the {2} environment variable is set properly. You may need to run once as an admin to ensure all resources are extracted. Details:{0} {1}".FormatWith(Environment.NewLine, ex.ToString(), EnvironmentVariables.System.ChocolateyInstall));

```

Comment:
Why is the `FormatWith` needed here in the `Error` method, but not in the `Warn` method above?